### PR TITLE
Added a "plant DNA manipulator" to Botany backroom

### PR DESCRIPTION
**Pull request by coiax**
On bobdobbington's request, this map change is made.

:cl: coiax, bobdobbington, Core0verload
rscadd: An experimental plant DNA manipulator, based on machinery recovered from ancient seed vaults, has been installed in the Botany department.
/:cl:

> 7:22:23 PM bobdobbington  so, not too long ago coreoverload did a huge
> overhaul and refactor of how plants work
> 7:22:48 PM bobdobbington  breaking out each interesting mutant trait
> into its own gene
> 7:23:10 PM bobdobbington  so, like, the slipperiness of bananas, the
> fluacid content of deathnettles, things like that
> 7:23:34 PM bobdobbington  The idea was to be able to crossbreed
> different types of plants
> 7:23:52 PM bobdobbington  So you could have like say grass tiles that
> glowed like glowshrooms
> 7:24:19 PM bobdobbington  Or glowshrooms that could slip
> 7:24:36 PM bobdobbington  This is the machine that enables the
> manipulation of those traits, which already exist in the code

### CODE REVIEW

**Comment by Supermichael777** [comment]
its work in progresses and horribly op.

**Comment by LatD** [comment]
How about adding it to lavaland botany? There people could test it and play with it freely and not affecting the round.

**Comment by coiax** [comment]
Literally made this PR because bobdobbington asked me to.

And that's not a bad idea, Lati, although this is a modern machine, making it de-constructable and movable, and vault creatures are found occasionally by miners. Makes the chance of retrieval a lot lower, depends how op it is?

**Comment by Core0verload** [comment]
I coded this one, and I support this change. It needs live testing to determine how OP is it, and we can always nerf the fuck out of it if it becomes a problem.

Add a box of plant gene disks to botanists lockers too. The machine is useless without the disks.

**Comment by coiax** [comment]
Can you suggest some sane research levels?

**Comment by Core0verload** [comment]
Biology 3 and Programming 4.

**Comment by Cheridan** [comment]
>How about adding it to lavaland botany? There people could test it and play with it freely and not affecting the round.

Let's have the actual botanists have something nice for a change instead of giving all their toys to roundstart observers

**Comment by coiax** [comment]
@Cheridan too late, already added it the seed vault.

**Comment by KorPhaeron** [comment]
@Cheridan 

It's in both

File: code/modules/research/designs/machine_designs.dm [hunk 1]
```diff
@@ -471,4 +471,14 @@
...
 	build_path = /obj/item/weapon/circuitboard/grounding_rod
+	category = list ("Misc. Machinery")
+
+/datum/design/plantgenes
+	name = "Machine Design (Plant DNA Manipulator Board)"
+	desc = "The circuit board for a grounding rod."
+	id = "plantgenes"
+	req_tech = list("programming" = 4, "biotech" = 3)
+	build_type = IMPRINTER
+	materials = list(MAT_GLASS = 1000, "sacid" = 20)
```
Comment (KorPhaeron) [inline code/modules/research/designs/machine_designs.dm]: This is basically free, but I guess these are placeholder/test values anyway

File: code/modules/research/designs/machine_designs.dm [hunk 2]
```diff
@@ -471,4 +471,14 @@
 	build_type = IMPRINTER
 	materials = list(MAT_GLASS = 1000, "sacid" = 20)
 	build_path = /obj/item/weapon/circuitboard/grounding_rod
+	category = list ("Misc. Machinery")
+
+/datum/design/plantgenes
+	name = "Machine Design (Plant DNA Manipulator Board)"
+	desc = "The circuit board for a grounding rod."
```
Comment (KorPhaeron) [inline code/modules/research/designs/machine_designs.dm]: Wrong desc

File: code/modules/research/designs/machine_designs.dm (hunk 1 above)
Comment (coiax) [inline code/modules/research/designs/machine_designs.dm]: It's a circuit board, the cost is standardised.

**Pull request merged**